tools/src/main/scala/scala/scalanative/io/Classpath.scala

```diff
@@ -0,0 +1,55 @@
+package scala.scalanative.io
+
+import java.net.URI
+import java.nio.ByteBuffer
+import java.nio.file._
+
+import scala.collection.JavaConverters._
+
+/**
+  * A values which represents a member of a classPath.
+  */
+sealed abstract class Classpath[T <: VFile] {
```

Comment:
Given that `Classpath` is only used by linker, it should stay in `linker` package.